python/hail/methods/io.py

```diff
@@ -86,6 +87,94 @@ def export_gen(dataset, output, precision=4):
         
     Env.hail().io.gen.ExportGen.apply(dataset._jvds, output, precision)
 
+@handle_py4j
+# @require_biallelic
```

Comment:
you can rewrite this as `base = require_biallelic(base, 'export_plink')` right before you pass it into the `ExportPlink` function.